docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+services:
+  
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    env_file: 
+      - .env
+    environment:
+      - FLASK_APP=app.factory:create_app('production')
```

Comment:
can we change this to non-prod there are some niceties in flask when you are running in dev mode.